File: Application/Services/Operations/Main/Partners/Dtos/PartnerDto.cs
using System;
using System.Collections.Generic;
using Application.Services.Operations.Main.Inheritances;
using Application.Services.Operations.Main.Partners.Enums;
using Application.Services.Operations.Outsourced.Dtos;
using Domain.Entities.Main;

namespace Application.Services.Operations.Main.Partners.Dtos
{
    public class PartnerDto : MainEntitiesBaseDto
    {

        public List<PaymentDataDto> PaymentsDataDto {get; set;}
        public PartnerBusinessEnumDto PartnerBusiness { get; set; }
        public List<CollectDeliverDto> CollectDeliversTransporters { get; set; }
        public virtual List<ElectronicRepairDto> ElectronicsRepairs { get; set; }
    }
}

File: Views/src/components/main/partner/dtos/partner-dto.ts


import { MainEntitiesBase } from "../../inheritances/dtos/main-entities-base";
import { PartnerBusinessEnumDto } from "../../../main/partner/commons-components/dtos/enums/partner-business-enum-dto";
import { PaymentDataDto } from "../../../main/partner/commons-components/dtos/payment-data-dto";



export class PartnerDto extends MainEntitiesBase{
  paymentsData: PaymentDataDto;
  partnerBusiness: PartnerBusinessEnumDto;
  // collectDeliversTransporters: CollectDeliverDto[];
  // electronicsRepairs: ElectronicRepairDto[];
}
